### PR TITLE
Map application command models and routes

### PR DESCRIPTION
**Pull request by AnotherZane**

### CODE REVIEW

File: src/Disqord.Rest.Api/Routing/Route.Static.cs [hunk 1]
```diff
@@ -319,11 +327,19 @@ public static class Interactions
...
             public static readonly Route DeleteInitialResponse = Delete("webhooks/{0:application_id}/{1:interaction_token}/messages/@original");
 
-            public static readonly Route CreateFollowupResponse = Post("webhooks/{0:application_id}/{1:interaction_token}");
+            public static readonly Route CreateFollowupMessage = Post("webhooks/{0:application_id}/{1:interaction_token}");
 
-            public static readonly Route ModifyFollowupResponse = Patch("webhooks/{0:application_id}/{1:interaction_token}/messages/{2:message_id}");
+            public static readonly Route ModifyFollowupMessage = Patch("webhooks/{0:application_id}/{1:interaction_token}/messages/{2:message_id}");
 
-            public static readonly Route DeleteFollowupResponse = Delete("webhooks/{0:application_id}/{1:interaction_token}/messages/{2:message_id}");
+            public static readonly Route DeleteFollowupMessage = Delete("webhooks/{0:application_id}/{1:interaction_token}/messages/{2:message_id}");
```
Comment (Quahu) [inline src/Disqord.Rest.Api/Routing/Route.Static.cs:334]: Don't rename these. Response is better, as it's not always going to be a message. E.g. deferred message update in components.

File: src/Disqord.Rest.Api/Content/Json/ModifyApplicationCommandPermissionsJsonRestRequestContent.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using Disqord.Models;
+using Disqord.Serialization.Json;
+
+namespace Disqord.Rest.Api
+{
+    public class ModifyApplicationCommandPermissionsJsonRestRequestContent : JsonModelRestRequestContent
+    {
+        [JsonProperty("id")] 
+        public Optional<Snowflake> Id;
```
Comment (Quahu) [inline src/Disqord.Rest.Api/Content/Json/ModifyApplicationCommandPermissionsJsonRestRequestContent.cs:9]: Shouldn't this be required?

**Review by Quahu — CHANGES_REQUESTED** [review]
Have you tested out all the endpoints? It's a bit difficult to judge if the endpoints and contents are mapped correctly because of Discord's docs.

File: src/Disqord.Rest.Api/Routing/Route.Static.cs (hunk 1 above)
Comment (AnotherZane) [reply src/Disqord.Rest.Api/Routing/Route.Static.cs]: Reverted

File: src/Disqord.Rest.Api/Content/Json/ModifyApplicationCommandPermissionsJsonRestRequestContent.cs (hunk 2 above)
Comment (AnotherZane) [reply src/Disqord.Rest.Api/Content/Json/ModifyApplicationCommandPermissionsJsonRestRequestContent.cs:9]: Discussed in the discord (Needs to be optional as it's reused for another endpoint)

**Review by Quahu — APPROVED** [review]

**Pull request merged**